baby-gru/src/components/BabyGruMap.js

```diff
@@ -141,7 +152,7 @@ BabyGruMap.prototype.clearBuffersOfStyle = function (gl, style) {
     //Empty existing buffers of this type
     $this.displayObjects[style].forEach((buffer) => {
         buffer.clearBuffers()
-        gl.displayBuffers = gl.displayBuffers.filter(glBuffer => glBuffer.id !== buffer.id)
+        gl.displayBuffers = gl.displayBuffers?.filter(glBuffer => glBuffer.id !== buffer.id)
```

Comment:
This is the bit of `BabyGruMap` that I had to change. When I used `this.clearBuffersOfStyle(gl, displayObject)` I got a `gl.displayBuffers is undefined` error message.